modules/kafka/kafka-dest-driver.c

```diff
@@ -273,25 +285,19 @@ _kafka_delivery_report_cb(rd_kafka_t *rk,
                           void *opaque, void *msg_opaque)
 {
   KafkaDestDriver *self = (KafkaDestDriver *) opaque;
-  LogMessage *msg = (LogMessage *) msg_opaque;
 
   /* we already ACKed back this message to syslog-ng, it was kept in
    * librdkafka queues so far but successfully delivered, let's unref it */
 
   if (err != RD_KAFKA_RESP_ERR_NO_ERROR)
     {
-      LogThreadedDestWorker *worker = (LogThreadedDestWorker *) self->super.workers[0];
-      LogQueue *queue = worker->queue;
-      LogPathOptions path_options = LOG_PATH_OPTIONS_INIT_NOACK;
-
       msg_debug("kafka: delivery report for message came back with an error, putting it back to our queue",
```

Comment:
minor stuff, but still: Can you reword the debug message too, please? :)
It states that we re-add the message to the queue of syslog-ng.

Reply:
Sure!